### PR TITLE
Expand and clarify sshd README.md. Correct typos.

### PR DESCRIPTION
**Pull request by bc1121**
Added ~/ to indicate the global .ddev folder level that was being affected.

Added ~/ to clarify and allow functional cut and paste of the command.

Added command to rename id_rsa.pub to authorized keys, to clarify and complete the necesscary work, and allow one cut and paste to start and finish the work.

Clarified for user that the project .ddev/homeadditions/ folder could be used as well, because it was a stumbling block for me and knowledge gained along the way. At first I was manually performing the mkdir, cp, mv operation at the project level and it worked but I had to look closer to see the command was written for the global(user?) level.

## The New Solution/Problem/Issue/Bug:
It was not clear at first that the "easy" way was targeting the global (single user level?) .ddev folder instead of the project level .ddev folder files.
There was a typo (missing ~/) that only allowed the command to work in the users home directory.
Also I had to know to rename the id_rsa.pub file to authorized_keys. The instructions/"easy command" felt incomplete.
I feel the updates will enable users at my level and newer to use this feature more easily.

## How this PR Solves The Problem:
Clarifies which level .ddev folder is being targeted.
Fixes typo that allows cut and paste of existing command to run from any folder.
Modifies the command to finish the necesscary work and shows the user the final step for success as well as giving them a cut and paste command that runs from anywhere.

## Manual Testing Instructions:
Just follow the directions on list item # 2 of the [README.md](https://github.com/drud/ddev-contrib/tree/master/recipes/sshd) page.

Try it first as is then try it with the new instructions.

You may have to clear any .ssh folders that were initially created to cleanup and reset the experiment between tries.

Thanks for your work on DDEV-Local. It's great!

### CODE REVIEW

**Comment by rfay** [comment]
You want both of today's PRs to be live right? They don't relate to each other?

**Comment by bc1121** [comment]
Yes, they are both good in my opinion and are separate.  Interested in feedback. I kept them separate because they affect different files. The yaml file update being the most critical I think and more cut and dry.

The updates to the readme were more subjective and the wording could be debated so I kept the script bug in the yaml file separate to it could slide right in without being held up by the wording in the readme.md.

File: recipes/sshd/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
-2. Authorize your ssh client to access the web container's ssh server by adding a `.ddev/homeadditions/.ssh/authorized_keys`, which will be copied into the ~/.ssh directory in the web container. The easiest way to do this is `mkdir -p ~/.ddev/homeadditions/.ssh && cp ~/.ssh/id_rsa.pub .ddev/homeadditions/.ssh`, assuming that your public ssh key is named id_rsa.pub and is in ~/.ssh.
+2. Authorize your ssh client to access the web container's ssh server by adding a global `~/.ddev/homeadditions/.ssh/authorized_keys`, which will be copied into the ~/.ssh directory in the web container. The easiest way to do this is `mkdir -p ~/.ddev/homeadditions/.ssh && cp ~/.ssh/id_rsa.pub ~/.ddev/homeadditions/.ssh && mv ~/.ddev/homeadditions/.ssh/id_rsa.pub ~/.ddev/homeadditions/.ssh/authorized_keys`, assuming that your public ssh key is named id_rsa.pub and is in ~/.ssh.<br />You can also create the authorized_keys file in the projectname/.ddev/homeadditions/.ssh folder on a project by project basis but this command takes care of authorized_keys for all of your projects at once and can be executed from any directory.
```
Comment (rfay) [inline recipes/sshd/README.md:6]:
```suggestion
2. Authorize your ssh client to access the web container's ssh server by adding a global `~/.ddev/homeadditions/.ssh/authorized_keys`, which will be copied into the ~/.ssh directory in the web container. The easiest way to do this, assuming your ssh pubkey is ~/.ssh/id_rsa.pub, is 
    ```
    mkdir -p ~/.ddev/homeadditions/.ssh
    cp ~/.ssh/id_rsa.pub ~/.ddev/homeadditions/.ssh/authorized_keys
    ```
   (You can also create the authorized_keys file in the project `.ddev/homeadditions/.ssh` folder on a project by project basis but this command takes care of authorized_keys for all of your projects at once and can be executed from any directory.)
```

**Review by rfay — CHANGES_REQUESTED** [review]
Thanks for this!

**Pull request merged**